docssrc/src/permissions.md

```diff
@@ -117,3 +117,145 @@ Now we declare our command with arguments. We use a `PlayerArgument` and apply t
 > As you can see, there are multiple ways of applying permissions to commands with arguments. In the `/god` command shown above, the permission was applied to the whole command. In the `/kill` command shown above, the permission was applied to the argument.
 >
 > There's not really much difference between the two methods, but I personally would use _argument permissions_ as it has greater control over arguments.
+
+## Child Base Permissions
+### Example - /economy command with argument permissions
+
+For example, say we're registering a command `/economy`:
+
+```mccmd
+/economy                         - shows your own balance
+/economy <target>                - shows you another players balance
+/economy give  <target> <amount> - gives the target a set amount of money
+/economy reset <target> <amount> - resets the targets balance
+```
+
```

Comment:
Here, we also need to state the permissions for each command/subcommand before we jump straight into the code